scripts/start_integration_test_contracts_with_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Get the latest Finschia tag
+TAG=$(curl "https://api.github.com/repos/Finschia/finschia/tags" | jq -r '.[0].name')
+TAG=$(echo "$TAG" | cut -c 2-)
+
+TEST_DOCKER_IMAGE=finschia/finschianode:${TAG}
+docker pull finschia/finschianode:${TAG}
+
+export FNSAD="docker run -i --rm -p 26656:26656 -p 26657:26657 -v ${HOME}/.finschia:/root/.finschia ${TEST_DOCKER_IMAGE} fnsad"
+
+# Copy init_single.sh from Finschia/finschia/init_single.sh
+init_single=$(mktemp); curl "https://raw.githubusercontent.com/Finschia/finschia/v${TAG}/init_single.sh" > $init_single
```

Comment:
```suggestion
init_single=$(mktemp)
curl "https://raw.githubusercontent.com/Finschia/finschia/v${TAG}/init_single.sh" -o $init_single
```

Reply:
Why need add `v` before {TAG} ?

Reply:
Why not download init_single before `docker pull`, if so, it has no need to modify tag twice.

Reply:
Change to download init_single before `docker pull`. Thanks.
https://github.com/Finschia/finschia-wasm/pull/16/commits/04d15d8cf905073e1f113742d4aadd60a0bc6707